server/schedulers/evict_leader.go

```diff
@@ -231,7 +231,7 @@ func (s *evictLeaderScheduler) scheduleOnce(cluster opt.Cluster) []*operator.Ope
 		}
 		op, err := operator.CreateTransferLeaderOperator(EvictLeaderType, cluster, region, region.GetLeader().GetStoreId(), target.GetID(), operator.OpLeader)
 		if err != nil {
-			log.Debug("fail to create evict leader operator", zap.Error(err))
```

Comment:
Original error should be wrapped by "cause" in zap.

Reply:
done

Reply:
no need to modify it for now.